tests/lax_numpy_reducers_test.py

```diff
@@ -866,5 +866,19 @@ def testCumulativeSumBool(self):
                              dtype=jnp.bool_)
     np.testing.assert_array_equal(np.array([[True], [True], [False]]), out)
 
+  @jtu.sample_product(dtype=[dtypes.int4, dtypes.uint4, dtypes.int2, dtypes.uint2])
+  @jtu.ignore_warning(category=NumpyComplexWarning)
+  def test_low_bit_sum(self, dtype) -> None:
+    if dtype is not None:
```

Comment:
nit: let's do an explicit skip
```suggestion
    if dtype is None:
      self.skipTest(f"Current version of ml_dtypes does not support {dtype}")
```
And then put the rest of this in the main function block.

Reply:
I made the change